modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utils/iframes.js

```diff
@@ -13,44 +13,40 @@
  */
 
 import {OPEN_MODAL, OPEN_MODAL_FROM_IFRAME} from './eventsDefinitions';
+import {logError} from './logError';
 
-const iframeHandlerModalNamespace = 'iframe-handler-modal_';
-let modalsCounter = 0;
-const registeredModals = new Map();
+export function isPageInIframe() {
+	return window.location !== window.parent.location;
+}
 
-Liferay.on(OPEN_MODAL_FROM_IFRAME, (payload) => {
-	let firstAvailableModalId = null;
+const registeredModals = new Set();
+let modalsCounter = 0;
 
-	// eslint-disable-next-line no-for-of-loops/no-for-of-loops
-	for (const [modalId, modalRef] of registeredModals.entries()) {
-		if (modalRef) {
-			firstAvailableModalId = modalId;
-			break;
+if (!isPageInIframe()) {
+	Liferay.on(OPEN_MODAL_FROM_IFRAME, (payload) => {
+		if (!registeredModals.size) {
+			return logError('No registered modals founds');
```

Comment:
With this line we are implying that `logError` and event callback are returning some value. In reality, they are not. While this code will work fine, for clarity and readability this should be
```js
logError('No registered modals found.');

return;
```
Also, a grammar fix: 'founds' -> 'found.'

Reply:
Sorry! Just fixed :)